src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyFlowNode.cs

```diff
@@ -31,11 +32,14 @@ public DependencyFlowNode(string repository, string branch)
         public HashSet<string> OutputChannels { get; set; }
         public HashSet<string> InputChannels { get; set; }
 
+        [JsonIgnore]
```

Comment:
Are these needed after you remove you test generation code?

Reply:
Turns out, nope.